r/R/dataset-format.R

```diff
@@ -33,6 +33,10 @@
 #'   * "tsv", equivalent to passing `format = "text", delimiter = "\t"`
 #' * `...`: Additional format-specific options
 #'
+#'   `format = "csv"``:
```

Comment:
This should be integrated with the `format = "text"` section at L47 below (since "csv" actually is for any delimited text file and is an alias for "text" in this function). Also can you please add a crossref like `See [Codec] for supported compression libraries`?